### ISSUE TITLE
Accessibility: the provider top metadata view is formatted as a list

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#153]
The top metadata information is formatted as a list with 6 items, whereas it should be formatted as a list with 3 items, a definition list, or a table. This results in the readout of "list with 6 items". This may also be because there are divs inside the definition list, which is not correct as far as I know.

![image](https://github.com/user-attachments/assets/d89ced51-7d4d-439c-9ec2-d48006e4dbfa)

**Comment by damianstasik** [comment]
> This may also be because there are divs inside the definition list, which is not correct as far as I know.

Using divs in the definition list [is valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#wrapping_name-value_groups_in_div_elements), but I'll take a look how to make it more accessible.
